webapp/tests/forms/steps/lotse/test_pauschbetrag.py

```diff
@@ -229,10 +229,10 @@ def test_if_merkzeichen_given_then_get_pauschbetrag_returns_result_of_calculate_
             'person_a_disability_degree': 25,
             'person_a_has_merkzeichen_bl': True,
             'person_a_has_merkzeichen_tbl': True,
-            'person_a_has_merkzeichen_h': True
+            'person_a_has_merkzeichen_h': True,
+            'person_a_requests_pauschbetrag': 'yes',
         })
         form_data = MultiDict({
-            'person_a_requests_pauschbetrag': 'yes',
```

Comment:
Why did you change that? That is for all these tests? I think this might have to do with the issue that we do not set the post data in the test_request_context. Maybe let's revisit these tests

Reply:
that is for all the tests in this file

Reply:
Reverted and added new fixture